lua/orgmode/config/mappings/init.lua

```diff
@@ -2,95 +2,126 @@ local m = require('orgmode.config.mappings.map_entry')
 
 return {
   global = {
-    org_agenda = m.action('agenda.prompt', { opts = { buffer = false } }),
-    org_capture = m.action('capture.prompt', { opts = { buffer = false } }),
+    org_agenda = m.action('agenda.prompt', { opts = { buffer = false, desc = 'org agenda' } }),
+    org_capture = m.action('capture.prompt', { opts = { buffer = false, desc = 'org capture' } }),
   },
   agenda = {
-    org_agenda_later = m.action('agenda.advance_span', { args = { 1 } }),
-    org_agenda_earlier = m.action('agenda.advance_span', { args = { -1 } }),
-    org_agenda_goto_today = m.action('agenda.reset'),
-    org_agenda_day_view = m.action('agenda.change_span', { args = { 'day' } }),
-    org_agenda_week_view = m.action('agenda.change_span', { args = { 'week' } }),
-    org_agenda_month_view = m.action('agenda.change_span', { args = { 'month' } }),
-    org_agenda_year_view = m.action('agenda.change_span', { args = { 'year' } }),
-    org_agenda_quit = m.action('agenda.quit'),
-    org_agenda_switch_to = m.action('agenda.switch_to_item'),
-    org_agenda_goto = m.action('agenda.goto_item'),
-    org_agenda_goto_date = m.action('agenda.goto_date'),
-    org_agenda_redo = m.action('agenda.redo'),
-    org_agenda_todo = m.action('agenda.change_todo_state'),
-    org_agenda_clock_in = m.action('agenda.clock_in'),
-    org_agenda_clock_out = m.action('agenda.clock_out'),
-    org_agenda_clock_cancel = m.action('agenda.clock_cancel'),
-    org_agenda_set_effort = m.action('agenda.set_effort'),
-    org_agenda_clock_goto = m.action('clock.org_clock_goto'),
-    org_agenda_clockreport_mode = m.action('agenda.toggle_clock_report'),
-    org_agenda_priority = m.action('agenda.set_priority'),
-    org_agenda_priority_up = m.action('agenda.priority_up'),
-    org_agenda_priority_down = m.action('agenda.priority_down'),
-    org_agenda_toggle_archive_tag = m.action('agenda.toggle_archive_tag'),
-    org_agenda_set_tags = m.action('agenda.set_tags'),
-    org_agenda_deadline = m.action('agenda.set_deadline'),
-    org_agenda_schedule = m.action('agenda.set_schedule'),
-    org_agenda_filter = m.action('agenda.filter'),
-    org_agenda_show_help = m.action('org_mappings.show_help'),
+    org_agenda_later = m.action('agenda.advance_span', { args = { 1 }, { opts = { desc = 'org later' } } }),
```

Comment:
For all the actions which have `args`, you need to pass `opts` on the same table. Currently you are nesting opts an additional table

Reply:
Yeah, that's because I was copy-pasting from the ones that have no `args`. My bad, will fix.